services/drupal/web/themes/epa_theme/source/_patterns/07-pages/listing-pages/public-notices.yml

```diff
@@ -3,3 +3,144 @@ body_class: 'not-front'
 page_title: 'Search Public Notices'
 has_footer: false
 intro: '<p>Search public notices by typing keywords in the Search field below, or by checking one or more boxes beside the filters listed on the right side. If you see “Show more” beneath a set of filters, click it to display the full list of filters. See the <a href="#">Public Notices home page</a> for additional ways to find EPA public notices.</p>'
+collection:
+  -
+    url: '#'
+    show_calendar: false
+    image_src: false
+    heading: 'Opportunity to Comment on Notice of Discharge Requirements for Jabil, Incorporated'
+    description: 'This is the summary, which can contain <abbr title="Hyper Text Markup Language">HTML</abbr> markup. It should be 300 characters or less.'
+    more_info:
+      -
+        text: '<strong>Comments Due:</strong> May 15, 2020<br><strong>Location:</strong> Colorado, Rocky Mountains'
```

Comment:
@jbwagner Instead of using the `<br>` tag, we should add the location info as a new item in the `more_info` array.

```
    more_info:
      -
        text: '<strong>Comments Due:</strong> May 15, 2020'
      -
        text: '<strong>Location:</strong> Colorado, Rocky Mountains'
```